clustered/client/src/main/java/org/ehcache/clustered/client/internal/ClusterTierManagerClientEntityFactory.java

```diff
@@ -88,6 +88,13 @@ public void abandonLeadership(String entityIdentifier) {
     }
   }
 
+  public void abandonLeadershipIfLeader(String entityIdentifier) {
```

Comment:
There is a method called abandonLeadership above, but it is not used anywhere except tests..

Also there are methods that does localholds, but local holds are released using try-with-resources... so the problem is only when acquireLeadership is explicitly called.. a corresponding abandon must be issued.

Reply:
Should we keep an API method if it is used only in tests?